### PR TITLE
docs: Update README.md

### PR DESCRIPTION
**Pull request by pdelewski**
Unsufficient VirtualBox priviledges

##### Description

Fill in your description here.

---

##### Checklist

- [ ] Changelog updated

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: vagrant/README.md [hunk 1]
```diff
@@ -23,6 +23,22 @@ You can set up the Vagrant environment with just one command:
 ```bash
 vagrant up
 ```
+If you will experience following error
```
Comment (swiatekm-sumo) [inline vagrant/README.md:26]: This is only relevant to MacOS, and I think we should point this out.

**Review by swiatekm-sumo — COMMENTED** [review]
We use https://www.conventionalcommits.org/en/v1.0.0/ here, can you reformat your commit and PR title accordingly?

File: vagrant/README.md (hunk 1 above)
Comment (sumo-drosiek) [inline vagrant/README.md:26]: please add empty line between `->```` and text

Comment (pmalek-sumo) [inline vagrant/README.md:26]:
```suggestion

If you experience the following error:
```

Comment (pdelewski) [reply vagrant/README.md]: Done. Check if that's correct convention

Comment (pdelewski) [reply vagrant/README.md]: ok

Comment (pdelewski) [reply vagrant/README.md]: Updated

**Review by pmalek-sumo — APPROVED** [review]

**Review by sumo-drosiek — APPROVED** [review]

**Comment by pdelewski** [comment]
> We use https://www.conventionalcommits.org/en/v1.0.0/ here, can you reformat your commit and PR title accordingly?

done

**Comment by swiatekm-sumo** [comment]
> done

All of your commits and the PR title need to follow the convention for the CI to accept it. Just make the PR title the same as the commit message.

**Review by swiatekm-sumo — APPROVED** [review]

**Comment by pmalek-sumo** [comment]
@pdelewski I believe that in order to make "Semantic Pull Request" happy you need to change `doc:` to `docs:` as per https://www.conventionalcommits.org/en/v1.0.0/

**Pull request merged**